site/outline/outline.rst

```diff
@@ -333,8 +308,11 @@ violate an instructor's intellectual property rights and the Canadian Copyright
 be subject to disciplinary actions.
 
 
-Statement of Academic Offenses
-===============================
+Statement of Academic Offenses and Academic Integrity
+=====================================================
+
+Please ensure that you are aware of the policy on Academic Integrity. Details can be found
+`here <https://www.stfx.ca/media/40111/download>`_.
```

Comment:
Any statements in this about use of AI? Worth a section?

Reply:
The only reason I'm not is because our academic calendar says that of there is NO statement, it defaults to not being allowed to use it. I figure, as soon as I say something, someone will try to interpret my wording in the most annoying way possible.